### PR TITLE
Feature/prenotification bulk upload datafile validation

### PR DESCRIPTION
**Pull request by MichaelCoull1**
PBI 65554 (tasks 66474 and 66476) - completed validation checks of file size and file type.

Also, fixed the lack of bullet points on the upload-prenotifications page.

### CODE REVIEW

File: src/EA.Iws.Web/Web.config [hunk 1]
```diff
@@ -15,7 +15,7 @@
   </configSections>
   <system.web>
     <compilation debug="true" targetFramework="4.5.2" />
-    <httpRuntime targetFramework="4.5.2" enableVersionHeader="false" encoderType="System.Web.Security.AntiXss.AntiXssEncoder" maxRequestLength="51200" />
+    <httpRuntime targetFramework="4.5.2" enableVersionHeader="false" encoderType="System.Web.Security.AntiXss.AntiXssEncoder" maxRequestLength="2147483647" executionTimeout="600" />
```
Comment (MichaelCoull1) [inline src/EA.Iws.Web/Web.config]: @GonzaloGirasol - Worth being aware of the execution timeout property here. Set to 10mins (value is in seconds). Any concerns?

Comment (GonzaloGirasol) [reply src/EA.Iws.Web/Web.config]: Probably set this to 5 minutes. I feel like 10 is too long. But then again I cant' really say at this point, it's hard to judge how well it's going to perform...

Comment (GonzaloGirasol) [reply src/EA.Iws.Web/Web.config]: Actually on this note, just leave it as it is, we've got no idea at this stage..

File: src/EA.Iws.Web/Infrastructure/BulkUpload/BulkMovementValidator.cs [hunk 2]
```diff
@@ -28,19 +28,28 @@ private async Task<List<RuleResult<BulkMovementFileRules>>> GetFileRules(HttpPos
         {
             var rules = new List<RuleResult<BulkMovementFileRules>>();
             var fileExtension = Path.GetExtension(file.FileName);
+            int maximumFileSize = 2147483647; // 4GB
```
[the remark this answers is not in the document]
Comment (MichaelCoull1) [reply src/EA.Iws.Web/Infrastructure/BulkUpload/BulkMovementValidator.cs]: Done

**Pull request merged**